fe1-web/src/features/rollCall/components/RollCallClosed.tsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import React, { useCallback, useMemo } from 'react';
+import { useToast } from 'react-native-toast-notifications';
+
+import ScreenWrapper from 'core/components/ScreenWrapper';
+import { ToolbarItem } from 'core/components/Toolbar';
+import { Hash } from 'core/objects';
+import { FOUR_SECONDS } from 'resources/const';
+import STRINGS from 'resources/strings';
+
+import { requestReopenRollCall } from '../network';
+import { RollCall } from '../objects';
+import AttendeeList from './AttendeeList';
+import RollCallHeader from './RollCallHeader';
+
+const RollCallClosed = ({ rollCall, laoId, isOrganizer }: IPropTypes) => {
+  const toast = useToast();
+
+  const makeToastErr = useCallback(
+    (error: string) => {
+      toast.show(error, {
+        type: 'danger',
+        placement: 'bottom',
+        duration: FOUR_SECONDS,
+      });
+    },
+    [toast],
+  );
+
+  const onReopenRollCall = useCallback(() => {
+    // Once the roll call is opened the first time, idAlias is defined
+    if (rollCall.idAlias) {
+      requestReopenRollCall(laoId, rollCall.idAlias).catch((e) => {
+        makeToastErr(STRINGS.roll_call_location_error_reopen_roll_call);
+        console.debug(STRINGS.roll_call_location_error_reopen_roll_call, e);
+      });
+    } else {
+      makeToastErr(STRINGS.roll_call_location_error_reopen_roll_call_no_alias);
+      console.debug(STRINGS.roll_call_location_error_reopen_roll_call_no_alias);
```

Comment:
same comment about the guard clause, and `console.debug` should probably be a warn as well, right?